### DISCUSSION TITLE
Malicious or ransom inside windows exec ?

### DISCUSSION THREAD

**Discussion by jaaaaf** [vzhd1701/enex2notion#58] [General]
virustotal.com and 3 securitiy vendors found some malicious inside windows exec...

![image](https://user-images.githubusercontent.com/2169428/196489089-6315d0b8-4fc5-43be-8ba9-15eb8d7a6118.png)

**Comment by EwanMcP** [comment]
Down to one detection now (Elastic), and that's rated "moderate confidence". Nothing wrong with the exe, it's a false positive.

**Comment by MrSamoDude** [comment]
The latest Windows version appears to be flagged by 8 vendors.

<img width="724" alt="image" src="https://github.com/vzhd1701/enex2notion/assets/136279048/78665da7-9f33-491c-9cbc-1a1b5ee3e993">
